frontend/src/components/Auth/SignUp.js

```diff
@@ -74,32 +65,48 @@ class SignUp extends React.Component {
         }
 
         auth.createUserWithEmailAndPassword(email, password)
-            .then(() => {
+            .then((data) => {
                 console.log('New user created!')
-                this.setState({newUserCreate: true})
+                const newUserInfo = {
+                    uid: data.user.uid,
+                    email: this.state.email,
+                    password: this.state.password,
+                    firstName: this.state.firstName,
+                    lastName: this.state.lastName,
+                    contact: {phone: this.state.phoneNumber},
+                    paymentMethods: ['Venmo', 'Cash', 'Zelle'],
+                    school: 'School University'
```

Comment:
Are we not letting user input their university name?

Reply:
I'll update the signup form completely in my next pull request. This one is just for user authorization.